samples/nrf9160/agps/src/main.c

```diff
@@ -146,15 +148,29 @@ static void cloud_event_handler(const struct cloud_backend *const backend,
 
 			if (ret == 0) {
 				k_delayed_work_submit(&reboot_work, K_NO_WAIT);
+				break;
 			}
-			break;
 		}
 
 		int err = gps_process_agps_data(evt->data.msg.buf,
 						evt->data.msg.len);
 		if (err) {
 			LOG_INF("Unable to process agps data, error: %d", err);
 		}
+#if defined(CONFIG_AGPS_SINGLE_CELL_ONLY)
+		{
+			double lat, lon;
+
+			err = gps_get_last_cell_location(&lat, &lon);
+			if (err) {
+				LOG_ERR("Failed to get GPS data, error: %d", err);
+				break;
+			}
+
+			LOG_INF("\nLongitude:  %f\nLatitude:   %f\n", lat, lon);
+			gps_stop(gps_dev);
```

Comment:
if `CONFIG_AGPS_SINGLE_CELL_ONLY` is enabled there is no need to call `gps_start()`

Reply:
In this sample, gps_agps_request() is called from gps_handler(GPS_EVT_AGPS_DATA_NEEDED), so we still need to start GPS.
Sequence in the log:
I: CLOUD_EVT_READY
I: Service info sent to cloud
D: GPS mode is enabled
D: GPS operational
I: Periodic GPS search started with interval 240 s, timeout 120 s
I: GPS_EVT_SEARCH_STARTED
D: A-GPS data update needed
I: GPS_EVT_AGPS_DATA_NEEDED
D: Created A-GPS request: {"appId":"SCELL","messageType":"DATA","data":{"mcc":440,"mnc":20,"tac":4202,"eci":10720861,"phycid":0}}
D: A-GPS request sent

But SCELL does not really need GPS so in the sample project I made above, no GPS started.